.github/workflows/coding-standards.yml

```diff
@@ -1,41 +1,16 @@
-
 name: "Coding Standards"
 
 on:
   pull_request:
     branches:
       - "*.x"
-      - "master"
   push:
     branches:
       - "*.x"
-      - "master"
 
 jobs:
   coding-standards:
     name: "Coding Standards"
-    runs-on: "ubuntu-20.04"
-
-    strategy:
-      matrix:
-        php-version:
-          - "7.4"
-          - "8.0"
-
-    steps:
-      - name: "Checkout"
-        uses: "actions/checkout@v2"
-
-      - name: "Install PHP"
-        uses: "shivammathur/setup-php@v2"
-        with:
-          coverage: "none"
-          php-version: "${{ matrix.php-version }}"
-          tools: "cs2pr"
-
-      - name: "Install dependencies with Composer"
-        uses: "ramsey/composer-install@v1"
-
-      # https://github.com/doctrine/.github/issues/3
-      - name: "Run PHP_CodeSniffer"
-        run: "vendor/bin/phpcs -q --no-colors --report=checkstyle | cs2pr"
+    uses: "doctrine/.github/.github/workflows/coding-standards.yml@1.1.1"
+    with:
+      php-version: '7.4'
```

Comment:
You can omit these 2 lines, and it will run with 8.0 (and it will probably pass)

Reply:
I tried that, but it won't pass with 8.0, because of the whole setup not being installable under 8.0 without `--ingnore-platform-arg=php`. For the sake of simplicity, I am keeping coding standards at 7.4 currently. Both here and upstream in `DoctrineModule`.

Reply:
Ok great 👍